Cargo.toml

```diff
@@ -26,7 +26,7 @@ overflow-checks = true
 [dependencies]
 cosmwasm-std = { version = "=2.1.0", default-features = false }
 cw-storage-plus = "=2.0.0"
-provwasm-std = { git = "https://github.com/provenance-io/provwasm", tag = "v2.3.0-rc1" }
+provwasm-std = { git = "https://github.com/provenance-io/provwasm", tag = "v2.3.0" }
```

Comment:
Just FYI we can move these back to just be version number specifications now that we 2.3.0 is published on crates.io